src/screens/d-apps/d-apps.selectors.ts

```diff
@@ -0,0 +1,5 @@
+export enum DAppsSelectors {
+  integratedDApp = 'DApps/Integrated DApp',
+  othersDApps = 'DApps/Other DApps',
```

Comment:
element names are absent

Reply:
intergratedDApp is button, othersDApps is item